sys/kern/vm_object.c

```diff
@@ -55,43 +55,36 @@ void vm_object_add_page(vm_object_t *obj, off_t offset, vm_page_t *pg) {
   }
 }
 
-static void vm_object_remove_page_nolock(vm_object_t *obj, vm_page_t *page) {
-  page->offset = 0;
-  page->object = NULL;
-
-  TAILQ_REMOVE(&obj->list, page, obj.list);
-  vm_page_free(page);
-  obj->npages--;
-}
-
-void vm_object_remove_page(vm_object_t *obj, vm_page_t *page) {
-  SCOPED_MTX_LOCK(&obj->mtx);
-
-  vm_object_remove_page_nolock(obj, page);
-}
-
-void vm_object_remove_range(vm_object_t *object, off_t offset, size_t length) {
-  SCOPED_MTX_LOCK(&object->mtx);
+static void vmo_remove_pages_nolock(vm_object_t *obj, vm_offset_t offset,
+                                    size_t length) {
+  assert(mtx_owned(&obj->mtx));
```

Comment:
`vmo` vs `vm_object` - could you stick to one?

Reply:
Done.